ethstats/ethstats.go

```diff
@@ -277,7 +277,11 @@ func (s *Service) loop() {
 					// proxied validator should sign
 					channel = signCh
 				}
-				channel <- &statsPayload
+				select {
```

Comment:
add a TODO here. to signal this is a patch, a and we should do a proper clean up of this code